config/filter.d/postfix.conf

```diff
@@ -15,12 +15,12 @@ _daemon = postfix(-\w+)?/(?:submission/|smtps/)?smtp[ds]
 prefregex = ^%(__prefix_line)s(?:NOQUEUE: reject:|improper command pipelining) <F-CONTENT>.+</F-CONTENT>$
 
 failregex = ^RCPT from \S+\[<HOST>\]: 554 5\.7\.1 
-            ^RCPT from \S+\[<HOST>\]: 450 4\.7\.1 Client host rejected: cannot find your hostname, (\[\S*\]); from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>$
-            ^RCPT from \S+\[<HOST>\]: 450 4\.7\.1 : Helo command rejected: Host not found; from=<> to=<> proto=ESMTP helo= *$
+            ^RCPT from \S+\[<HOST>\]: 450 4\.7\.1 Client host rejected: cannot find your (reverse )?hostname, (\[\S*\]); from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>
+            ^RCPT from \S+\[<HOST>\]: 450 4\.7\.1 .*: Helo command rejected: Host not found; from=<\S*> to=<\S*> proto=\S+ helo=.*$
```

Comment:
Sorry, no chances for dual greedy catch-all: `.* ... .*$`.
I see below in the test-case why it may be expected, but IMHO could be solved different way.